README.md

```diff
@@ -158,6 +158,12 @@ If you are looking for the codebase of the current production version of GATK, p
   on a cluster or the cloud by building a Spark jar with `gradle installSpark` and passing the resulting jar in `build/libs/`
   directly to either `spark-submit` or `gcloud`.
 
+* To pass JVM arguments to GATK (non-spark), use `JAVA_OPTS` like in this example:
```

Comment:
depends how you're running them, JAVA_OPTS will work if you're not using spark-submit, if you're using spark submit you need to use 
```
--conf "spark.executor.extraJavaOptions=-XX:+PrintGCDetails" 
```
and 
```
--driver-java-options "-XX:+PrintGCDetails" 
```